src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bulldozerNumber;
```

Comment:
The field `bulldozerNumber` should have an access modifier to encapsulate the field properly. Consider using the `private` modifier to enforce encapsulation.